src/main/java/com/java/margarita/training/Menagerie.java

```diff
@@ -0,0 +1,55 @@
+package com.java.margarita.training;
+
+import com.java.margarita.training.abstracts.Animal;
+import com.java.margarita.training.interfaces.Flying;
+import com.java.margarita.training.model.*;
+
+import java.util.ArrayList;
+
+public class Menagerie {
+    public static void main(String[] args) {
+        Cat kity = new Cat();
+        Cat busea = new Cat("Busea", 2);
+        Cat bob = new Cat("Bob", 1, "orange");
+
+        Dog lucky = new Dog("Lucky", 2, "white");
+
+        Crab crab = new Crab();
+
+        Dolphin dolphin = new Dolphin("Star", 1);
+
+        Dove dove = new Dove("Kesha", 1);
+
+        Duck duck = new Duck("Veronica", 1, "grey");
+
+        bob.eat();
+        lucky.run();
+        crab.say();
+        dolphin.swim();
+        busea.walk();
+        duck.walk();
+        dove.sleep();
+
+        ArrayList<Animal> animale = new ArrayList<>();
```

Comment:
unde trebuie sa fie acestea ca sa reduc liniile de cod?

Reply:
am creat metode in clasa Menagerie si le-am chemat in metoda main ca sa reduc liniile de con din metoda main